README.md

```diff
@@ -1 +1,185 @@
-# Parser in Go for Tarantool types 
+# Tarantool tuples converter in Go
+## Table of contents
+* [Documentation](#documentation)
+  * [Mapper](#mapper)
+  * [Converter](#converter)
+  * [Convert to tarantool types](#convert-to-tarantool-types)
+    * [Example](#example)
+    * [String to nullable](#string-to-nullable)
+    * [String to any/scalar](#string-to-anyscalar)
+    * [Customization](#customization)
+## Documentation
+
+### Mapper
+`Mapper[S,T]` maps objects of type `S` into objects of type `T`. Mappers are
+basic entities on which converters are based.  
+Implementations of some mappers are available, for example, mappers from strings
+to go types.   
+Usage example:
+```golang
+// Basic mapper.
+mapper := tupleconv.MakeStringToBoolMapper()
+result, err := mapper.Map("true") // true <nil>
+
+// Function based mapper.
+fMapper := tupleconv.MakeFuncMapper(func(s string) (string, error) {
+    return s + " world!", nil
+})
+result, err = fMapper.Map("hello") // hello world! <nil>
```

Comment:
Still want to talk about naming. In most programming languages `map` is a function/method which transforms a container by applying some function[s] to each element. This what our `converter` does. As for me, current `converter`  is a `map` algo, but current `map` is real converter.
@oleg-jukovec what do u think?

Reply:
I'd like to see something similar to JavaFX Converter interface. It has two methods: `getAsObject` and `getAsString`. So I can use the same implementation for to and from conversions (as I asked previously in some comment). So I will not get en error if I've accidentally created two different 'mappers' for to and from conversions.

Reply:
Okay, let's rename:
1. Map, Mapper -> Convert, Converter
2. Convert, Converter -> Map, Mapper

Reply:
Renamed.

Reply:
https://postimg.cc/jWRnyYgF